examples/tensorflow_converter/protoc.sh

```diff
@@ -0,0 +1 @@
+protoc -I=..  --cpp_out=.. ../tensorflow/core/framework/*.proto
```

Comment:
This shell script is used for other header and source generating apart from the main .proto file of TensorFlow file. If we include all compiled headers and source files, there will be a lot of useless files which might still be useful for further implementation.